src/centralserver/admin-service/ui-system-test/src/intTest/java/org/niis/xroad/cs/test/ui/glue/CommonUiStepDefs.java

```diff
@@ -42,4 +48,20 @@ public void preparePage() {
                 + "      document.head.insertAdjacentHTML('beforeend', style);");
     }
 
+    @SuppressWarnings("checkstyle:MagicNumber")
+    @Step("Page is prepared to be loading tested")
+    public void loadingTesting() {
+
+        var chromedriver = (ChromeDriver) Selenide.webdriver().driver().getWebDriver();
```

Comment:
I'd propose to add a "load-test flag here. And then create before and after hooks that will set 2g value and restore it to default after the test.

Also, I would phrase it "Browser is set to {string} network speeds"